### DISCUSSION TITLE
Possible to fill the relatedWorkItems from a wiql/shared query

### DISCUSSION THREAD

**Discussion by jgelon** [rfennell/AzurePipelines#1904] [Q&A]
Is it possible to fill the relatedWorkItems list from a wiql or shared query statement?

As I am using this tool for creating release notes based on just a SharedQuery, but my stakeholders like to see the parent ticket as one of the properties. I can add everything in the queryWorkItems but then my release notes also show the parents as an item.

**Comment by rfennell** [comment]
Sorry no, without a code change to the task the parents of the WIQL based WI are not added to the relatedWorkItems.

You could as you suggest get all the parents via the WIQL, could you not put some filter on the foreach that lists the WI out to filter our the items you want from the parents (maybe looking at the link field with a custom Handlebars extension)?

**Reply by jgelon** [reply]
Possibly, have to figure out how that should work
